include/boost/geometry/geometries/concepts/check.hpp

```diff
@@ -126,6 +126,11 @@ struct check<Geometry, polyhedral_surface_tag, false>
     : detail::concept_check::check<concepts::PolyhedralSurface<Geometry> >
 {};
 
+template <typename Geometry>
+struct check<Geometry, polyhedral_surface_tag, true>
+    : detail::concept_check::check<concepts::ConstPolyhedral<Geometry> >
```

Comment:
Since the source code requires C++14, the extra space between `>` and `>` is no longer required and should not be cultivated in new code

Reply:
will remove the extra space...